de/privacy/current.md

```diff
@@ -69,25 +69,43 @@ Diese Website nutzt Funktionen des Webanalysedienstes Google Analytics. Anbieter
 
-Du kannst die Erfassung durch Google Analytics verhindern, indem Du auf den folgenden Link klickst. Es wird ein Opt-Out-Cookie gesetzt, der die zukünftige Erfassung Deiner Daten beim Besuch dieser Website verhindert. Dieser Cookie gilt nur für diesen Browser und ausschließlich für den Besuch unserer Webseite und muss im Falle eines Löschens neu gesetzt werden: [Google Analytics deaktivieren](__JS_GOOGLE_ANALYTICS_DEACTIVATE__). Du kannst darüber hinaus Google Analytics mittels eines Browser Add-ons deaktivieren und damit das Erfassen und Verarbeiten Deiner Daten durch Google verhindern, wenn Du dies nicht wünschst. Dieses Bowser Add-on kannst Du hier herunterladen: [http://tools.google.com/dlpage/gaoptout?hl=de](https://tools.google.com/dlpage/gaoptout?hl=de).
+Du kannst die Erfassung Deiner Daten durch Google Analytics verhindern, indem Du auf den folgenden Link klickst. Es wird ein Opt-Out-Cookie gesetzt, der die zukünftige Erfassung Deiner Daten beim Besuch dieser Webseite verhindert. Dieser Cookie gilt nur für diesen Browser und ausschließlich für den Besuch unserer Webseite und muss im Falle eines Löschens neu gesetzt werden: [Google Analytics deaktivieren](__JS_GOOGLE_ANALYTICS_DEACTIVATE__). Du kannst darüber hinaus Google Analytics mittels eines Browser Add-ons deaktivieren und damit das Verarbeiten Deiner Daten durch Google verhindern, wenn Du dies nicht wünschst. Dieses Bowser Add-on kannst Du hier herunterladen: [http://tools.google.com/dlpage/gaoptout?hl=de](https://tools.google.com/dlpage/gaoptout?hl=de).
 
 Die Datenschutzrichtlinien von Google findest Du [hier](https://policies.google.com/?hl=de). Weitere Informationen dazu, wie Google Ihre Daten verarbeitet findest Du [hier](https://support.google.com/analytics/answer/6004245?hl=en&ref_topic=2919631).
 
+Die Benutzung erfolgt auf Grundlage von Art. 6 Abs. 1 lit. f. DSGVO. Unser berechtigtes Interesse an der Analyse des Nutzerverhaltens liegt in der Ermöglichung der Verbesserung unserer Websangebote.
+
+#### Matamo
+
+Auf einigen unserer Webseiten verwenden wir Matomo, eine Open‑Source‑Software zur statistischen Analyse der Zugriffe durch Besucher. Die Daten werden dabei ausschließlich durch uns selbst verarbeitet und gespeichert. Folgenden Daten werden dabei verarbeitet:
+
+* den Browser-Typ/-Version,
+* das verwendete Betriebssystem,
+* die Referrer-URL (die zuvor besuchte Seite),
+* die IP-Adresse in anonymisierter Form
+* Sprach- und Ländereinstellungen
+* Anzahl und Verweildauer der Besuche
+* auf der Website betätigten Links
+* die Uhrzeit der Serveranfrage
+* Die von Matomo gespeicherten Daten werden nicht auf andere Server übertragen oder an Dritte weitergegeben.
+
+Matomo ist auf unserer Webseite so konfiguriert, dass eine im Browser vorgenommene »Do‑Not‑Track«-Einstellung beachtet wird. Wenn Du mit der Speicherung und Auswertung Deiner personenbezogenen Daten nicht einverstanden bist, aktiviere die »Do‑Not‑Track«-Einstellung Deines Browsers.
```

Comment:
The used quotation marks should be consistent (see e.g. "demografische Merkmale")